tools/target.py

```diff
@@ -15,18 +15,35 @@ def register(command):
         "target", nargs="?", choices=platform.list(), help="the platform to select as the default platform"
     )
 
+    command.add_argument(
+        "-l",
+        "--local-build",
```

Comment:
We could probably just call this `--build`, since pulling + tagging is not actually building